tests/integration/commands/test_init.py

```diff
@@ -64,8 +64,17 @@ def fake_template_dirs(tmp_path):
 )
 @pytest.mark.parametrize("project_dir", [None, "project-dir"])
 @pytest.mark.usefixtures("fake_template_dirs")
-def test_init(app, capsys, monkeypatch, profile, expected_file, project_dir):
+def test_init(
+    app,
+    capsys,
+    monkeypatch,
+    profile,
+    expected_file,
+    project_dir,
+    empty_working_directory,
+):
     """Initialise a project."""
+    monkeypatch.chdir(empty_working_directory)
```

Comment:
empty_working_directory should already  do `monkeypatch.chdir` if I remember correctly